build.gradle

```diff
@@ -6,9 +6,9 @@ import org.jetbrains.kotlin.gradle.dsl.KotlinVersion
 
 plugins {
     id 'base'
-    id 'com.github.spotbugs' version '6.0.7' apply false
-    id 'org.jetbrains.kotlin.jvm' version '2.0.0' apply false
-    id 'com.google.devtools.ksp' version '2.0.0-1.0.24' apply false
+    id 'com.github.spotbugs' version '6.0.7'
+    id 'org.jetbrains.kotlin.jvm' version '2.0.0'
+    id 'com.google.devtools.ksp' version '2.0.0-1.0.24'
```

Comment:
`apply false` was a bit stupid considering we apply these plugins in all projects like 10 lines below